AssetEditor/ViewModels/MainViewModel.cs

```diff
@@ -296,6 +296,40 @@ public void CreateEmptyEditor(IEditorViewModel editorView)
             CurrentEditorsList.Add(editorView);
             SelectedEditorIndex = CurrentEditorsList.Count - 1;
         }
+
+        public bool AllowDrop(IEditorViewModel node, IEditorViewModel targeNode = default, bool insertAfterTargetNode = default)
```

Comment:
I am a huge fan of the new shorthand for this:
public bool func(args..) => true;